tests/lru_cache_test.py

```diff
@@ -44,30 +44,33 @@ def tearDown(self):
     self.name = None
     super().tearDown()
 
+  def assertCacheKeysEqualTo(self, keys):
```

Comment:
Nit: `assertCacheKeysEqual` to match the naming of other methods; or even `assertCacheKeys`.

Reply:
Done